src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,23 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    private static final String CHAR_MASK = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] words;
+        try {
+            words = Files.readString(Path.of(fileName)).toLowerCase().split("\\W+");
+        } catch (IOException e) {
+            throw new RuntimeException("No file");
```

Comment:
let's pass exception object that we have catched to RuntimeException constructor